manim/mobject/coordinate_systems.py

```diff
@@ -327,21 +338,39 @@ def get_lines(self):
         lines2 = VGroup(*x_lines2, *y_lines2)
         return lines1, lines2
 
-    def get_lines_parallel_to_axis(self, axis1, axis2, freq, ratio):
-        line = Line(axis1.get_start(), axis1.get_end())
+    def get_lines_parallel_to_axis(self, axis_parallel_to, axis_perpendicular_to, freq, ratio):
+        """Generated a set of lines parallel to an axis.
+
+        Parameters
+        ----------
+        axis_parallel_to : :class:`~.Line`
+            The axis with which the lines will be parallel.
+
+        axis_perpendicular_to : :class:`~.Line`
+            The axis with which the lines will be perpendicular. 
+
+        Returns
+        -------
+        lines1 : :class:`~.VGroup`
+            First set of lines (i.e the non faded lines parallel to 'axis_parallel_to')
+
+        lines2 : :class:`~.VGroup`
+            Second set of lines (i.e the faded lines parallel to 'axis_parallel_to')            
```

Comment:
Only one return value, and no names
Note that the usage of \` \` produces `this` on the sphinx docs result, so you might want to use that when referring to variables or parameters
```suggestion
        Tuple[:class:`~.VGroup`, :class:`~.VGroup`]
            The first (i.e the non faded lines parallel to `axis_parallel_to`) and second (i.e the faded lines parallel to `axis_parallel_to`) sets of lines, respectively.     
```